src/PySXO/workflow.py

```diff
@@ -1,8 +1,51 @@
-from .core.base import Base
+import json
+
 from attrdict import AttrDict
+from typing import Union, Dict, List
+
+from .core.base import Base
 
 class Workflow(Base):
```

Comment:
Add name property or some easy way to obtain workflow name, It is needed when we send message to Webex